doc/content/getting-started/packet-broker/_index.md

```diff
@@ -57,6 +57,25 @@ Localization | O | | Gateway locations, timestamps and signal quality
 >2. With Home Network that has `NetID` `000013` and tenant ID `example-com`: only join-requests, join-accepts and uplink and downlink MAC and application payload messages. This would mean to not forward the metadata.
 >3. A default policy (for all other networks): only uplink MAC and application payload messages. That would mean to not forward join-requests and join-accepts, and to not forward the metadata.
 
+### Gateway Visibilities
+
+Gateway visibility configuration allows network administrators to configure who sees which information about gateways. Having gateways listed publicly on the map can help discovering partner networks, but it may also lead to privacy concerns. Therefore, Packet Broker allows configuring gateway visibilities with fine-grained settings.
```

Comment:
```suggestion
Gateway visibility configuration allows network administrators to configure who sees which gateway-related information. Having gateways listed publicly on the map can help discovering partner networks, but it may also lead to privacy concerns. Therefore, Packet Broker allows configuring gateway visibility with fine-grained settings.
```